docs/resources/eks_cluster.md

```diff
@@ -123,6 +123,7 @@ resource "rafay_eks_cluster" "ekscluster-basic-with-ipv6" {
       type              = "eks"
       blueprint         = "minimal"
       cloud_provider    = "aws"
+      cross_account_role_arn = "arn:aws:iam::xxxxxxxxxxx:role/cross-account-role"
```

Comment:
Can we move this config under Advanced EKS cluster config with existing VPC & IAM example or add a new example for cross account role arn?

Reply:
I have the updated it @hardik-rafay